h2o-automl/src/main/java/water/automl/api/schemas3/AutoMLBuildSpecV99.java

```diff
@@ -163,6 +164,37 @@ public AlgoProvider() {
     }
   }
 
+  public static final class ScopeProvider implements ValuesProvider {
+    private static final String ANY_ALGO = "any";
+    private static final String[] SCOPES = ArrayUtils.append(new String[]{ANY_ALGO}, new AlgoProvider().values());
+
+    @Override
+    public String[] values() {
+      return SCOPES;
+    }
+  }
+
+  public static final class AutoMLCustomParameterV99<V> extends Schema<Iced, AutoMLCustomParameterV99<V>> {
+    @API(help="", valuesProvider=ScopeProvider.class, direction=API.Direction.INPUT)
```

Comment:
minor: no help for at all for all these fields? maybe they are too internal but this is conflicting  with the fact that APIs are for someone out there

Reply:
my bad, forgot to add it